api/v1alpha1/clusterconfig_types.go

```diff
@@ -246,6 +246,14 @@ type KubeadmClusterConfigSpec struct {
 	// KubeProxy defines the configuration for kube-proxy.
 	// +kubebuilder:validation:Optional
 	KubeProxy *KubeProxy `json:"kubeProxy,omitempty"`
+
+	// MaxParallelImagePullsPerNode defines the maximum number of parallel image pulls performed by each kubelet.
+	// If not set, the default value of 1 will be used.
+	// If set to 0, the maximum number of parallel image pulls will be unlimited.
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Default=1
+	MaxParallelImagePullsPerNode *int32 `json:"maxParallelImagePullsPerNode,omitempty"`
```

Comment:
I may be wrong, but wouldn't having `+kubebuilder:validation:Default=1` in there means that its rendered in the variable?
Is that what we want? With the rest of the APIs the convention here is to keep it pretty minimal.
```suggestion
	// MaxParallelImagePullsPerNode defines the maximum number of parallel image pulls performed by each kubelet.
	// If not set, the default value of 1 will be used.
	// If set to 0, the maximum number of parallel image pulls will be unlimited.
	// +kubebuilder:validation:Optional
	// +kubebuilder:validation:Minimum=0
	MaxParallelImagePullsPerNode *int32 `json:"maxParallelImagePullsPerNode,omitempty"`
```

Reply:
Removing the default can also simplify the handler

Reply:
Removing the default won't change the handler because it will still need to handle being explicitly set to 1, which should still result in no rollout if previously unset - the effective value would be the same.

Reply:
I'll remove it.